riscv/src/runtime.rs

```diff
@@ -235,20 +235,10 @@ impl Runtime {
         // Base syscalls
         r.add_syscall(
             Syscall::Input,
-            [
-                // TODO this is a quite inefficient way of getting prover inputs.
```

Comment:
Is this comment still valid?

Reply:
hmmm good catch, i think it is: we use 3 asm instructions to read an input value, maybe it can be improved later. I'll put the comment back